polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -2576,6 +2587,12 @@ BinOpInfo MLIRScanner::EmitBinOps(BinaryOperator *E, QualType PromotionType) {
   return {LHS, RHS, Ty, Opcode, E};
 }
 
+static void informNoOverflowCheck(LangOptions::SignedOverflowBehaviorTy SOB,
+                                  llvm::StringRef OpName) {
+  if (SOB != clang::LangOptions::SOB_Defined)
+    llvm::errs() << "Not emitting overflow-checked " << OpName << "\n";
```

Comment:
@victor-eds Should this be a warning or error? 
Is it safe to change to `mlirclang::warning`?

Reply:
Totally safe. I'll do it now.